### PR TITLE
Changelog - Update for 2.2.1

### PR DESCRIPTION
**Pull request by woodyrew**
Includes all merged code since previous release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,12 +3,22 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased]
+
+## [2.2.1]
 ### Changed
-* Updated dependencies
+* Updated dependencies ([#246])
+* Erroring test reading symbolic link to dir ([#229])
```
Comment (Ajedi32) [inline CHANGELOG.md]: If you check this PR, you'll see it actually includes more than a failing test; it also includes a fix for that test.

Comment (Ajedi32) [inline CHANGELOG.md]: So basically, Metalsmith will now traverse symbolic links in the source directory.

Comment (woodyrew) [inline CHANGELOG.md]: Ok, sorry I only read the code and the summary for that PR. We need a better process for this.

1. un-publish `2.2.1` from npm
2. correct changelog entry version
3. bump to `2.3.0`.
4. republish.

Agree? Did I miss anything?

Comment (Ajedi32) [inline CHANGELOG.md]: I don't think deprecating a version prevents NPM from automatically updating to it. Maybe try unpublishing that version?

Comment (Ajedi32) [inline CHANGELOG.md]: And yeah, process-wise I think it'd be good to start encouraging contributors to update the changelog in their PRs.

**Comment by Ajedi32** [comment]
Or alternately, publish v2.2.2 on a separate branch which reverts the
semver-minor change, then publish v2.3.0 with the minor change.

**Comment by woodyrew** [comment]
That sounds a better idea.

**Comment by woodyrew** [comment]
Actually, I should be able to add v2.2.2 to the same commit as v2.2.0 and publish.

**Pull request merged**